app/views/contexts/section_summary_context.py

```diff
@@ -48,8 +48,8 @@ def __call__(self, return_to: Optional[str] = "section-summary") -> Mapping:
             if isinstance(title_for_location, dict)
             else title_for_location
         )
-
-        page_title = self.get_page_title(title_for_location)
+        # Type ignore: expecting title_for_location not to be dict
+        page_title = self.get_page_title(title_for_location)  # type: ignore
```

Comment:
This is a valid error. `get_page_title` handle a `dict` `title_for_location`, the typing for it is incorrect. Can be fixed by:

```py
def get_page_title(self, title_for_location: Union[Mapping, str]) -> str:
```

Reply:
👍